packages/xo-web/src/common/select-objects.js

```diff
@@ -375,9 +389,33 @@ export const SelectHost = makeStoreSelect(
     const getHostsByPool = createGetObjectsOfType('host')
       .filter(getPredicate)
       .sort()
+      .groupBy('$pool')
+
+    const getPools = createGetObjectsOfType('pool')
+      .pick(
+        createSelector(
+          getHostsByPool,
+          hostsByPool => keys(hostsByPool)
+        )
+      )
+      .sort()
+
+    const compareContainers = createSelector(
```

Comment:
`compareContainers` should be a prop that you pass to the select so that every time you use a select, you're able to define a compare function to sort the containers.